ReactWindows/ReactNative.Shared.Tests/Bridge/JavaScriptModuleRegistryTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Newtonsoft.Json.Linq;
+using NMock;
+using NMock.Syntax;
+using NUnit.Framework;
+using ReactNative.Bridge;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Events;
+using System.Threading;
+
+namespace ReactNative.Tests.Bridge
+{
+    [TestFixture]
```

Comment:
According to @matthargett, they do if you're on Windows 10 (I'm currently running a Windows 7 VM, so a big thanks to him for confirming this for us) because we've added the NUnit3.TestAdapter NuGet.